Cargo.toml

```diff
@@ -11,13 +11,14 @@ readme = "README.md"
 [dependencies]
 futures = { version = "0.3", default-features = false }
 ring = { version = "0.16", default-features = false }
-rustls = { version = "0.21", default-features = false }
+rustls = { version = "0.22", default-features = false }
+rustls-pki-types = { version = "1", default-features = false }
```

Comment:
In rustls `0.22.1` you don't need to include `rustls-pki-types` dependency, as it's available as `rustls::pki_types`